tests/analyzers_manager/__init__.py

```diff
@@ -39,64 +44,140 @@ def get_params(cls):
 
     @classmethod
     def setUpClass(cls):
-        raise cls.skipTest("abstract test case")
+        if cls in [
+            _AbstractAnalyzersScriptTestCase,
+            _ObservableAnalyzersScriptsTestCase,
+            _FileAnalyzersScriptsTestCase,
+        ]:
+            return cls.skipTest(f"{cls.__name__} is an abstract base class.")
+        else:
+            return super(_AbstractAnalyzersScriptTestCase, cls).setUpClass()
 
     def setUp(self):
+        # cleanup
+        Job.objects.all().delete()
         # analyzer config
-        self.analyzer_config: dict = AnalyzerConfigSerializer.read_and_verify_config()
-        # define runtime configs
-        self.runtime_configuration = {
-            "Thug_URL_Info": {"test": True},
-            "Triage_Search": {"analysis_type": "submit"},
-        }
-        # save job
+        self.analyzer_configs = AnalyzerConfigSerializer.get_as_dataclasses()
+        return super().setUp()
+
+    def test_start_analyzers_all(self, *args, **kwargs):
+        print("\n[START] ---test_start_analyzers_all---")
+        print(
+            f"[REPORT] Job:{self.test_job.pk}, status:'{self.test_job.status}', analyzers:{self.test_job.analyzers_to_execute}"
+        )
+
+        # execute analyzers
+        celery_app.send_task(
```

Comment:
I am not sure what happens with `celery_app.send_task(` but surely celery is not executed. We could just execute the function directly without involving celery

Reply:
the `start_analyzers` function inside fires a celery task for each analyzer, so the problem will still persist.